crates/saya/core/src/verifier/starknet.rs

```diff
@@ -23,7 +23,7 @@ pub async fn starknet_verify(
         }])
         .send_with_cfg(&txn_config)
         .await
-        .unwrap();
+        .expect("Failed to send `verify_and_register_fact` transaction.");
```

Comment:
Would be better to return a result and let the caller handle possible retry.

Reply:
done